src/components/Dashboard/FilingHistoryList/filings/ContinuationIn.vue

```diff
@@ -122,6 +162,14 @@ export default class ContinuationIn extends Vue {
     return 'Unknown Name'
   }
 
+  mounted (): void {
+    if (this.tempRegNumber) {
+      // expand bootstrap filing by default
+      // assumes this the only filing in the Filing History list (which it should be)
+      this.setPanel(0)
+    }
```

Comment:
I'm thinking we could do this for all bootstrap filings (amalgamations, incorporations and registrations). Thoughts? Maybe next time I'm in this area of code.